docs/agent/kits/creating-kit-scripts.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Creating Kit Scripts
+read_when:
+  - "creating kit scripts"
+  - "adding Python CLI commands to kits"
+  - "testing kit script commands"
+---
+
+# Creating Kit Scripts
+
+Kit scripts are Python Click commands that extend erk's functionality through the kit system.
+
+## File Structure
+
+Kit scripts live in the kit's scripts directory:
+
+```
+packages/dot-agent-kit/src/dot_agent_kit/data/kits/<kit-name>/scripts/<kit-name>/<script_name>.py
+```
+
+Example: `packages/dot-agent-kit/src/dot_agent_kit/data/kits/erk/scripts/erk/get_closing_text.py`
+
+## Script Template
+
+```python
+#!/usr/bin/env python3
+"""Brief description of what the script does.
+
+Usage:
+    erk kit exec <kit> <command-name>
+
+Output:
+    Describe the output format
+
+Exit Codes:
+    0: Success
+    1: Error (if applicable)
+"""
+
+from pathlib import Path
+
+import click
+
+
+@click.command(name="command-name")
+def command_name() -> None:
+    """Click command docstring."""
+    # Implementation
+    click.echo("output")
+```
```

Comment:
always show click content. we want to encourage use of DI to achieve testability

Reply:
Resolved via /erk:pr-address at 2025-12-10 02:03 PM EST

_Addressed via `/erk:pr-address` at 2025-12-10 14:03 EST_